### PR TITLE
Adding things for Lusternia rifts, pathfind, wildnodes, and manse linking.

### PR DESCRIPTION
**Pull request by KaliLusternia**
Lets it automatically record special exits for TRANSVERSE and PATHFIND in lusternia, and adds a few aliases specific to Lusternia, letting you
- Link all astral nodes
- use the Astral jumping boots artifact
- Easily link manses to the aetherplex and cities
- lock PATHFIND exits if you do not have the skill.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -8155,6 +8342,183 @@ end
                             <string>mmapper went outside</string>
                         </eventHandlerList>
                     </Script>
+                    <Script isActive="yes" isFolder="no">
+                        <name>Astral utilities</name>
+                        <packageName></packageName>
+                        <script>local astralExits = {
+	{5079,5175,&quot;n&quot;},
```
Comment (keneanung) [inline mudlet-mapper.xml]: To make it easier for people to maintain the script, you may want to consider using dictionaries here with well-named keys.

Future you will thank you :P

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -8155,6 +8342,183 @@ end
...
+	s=&quot;n&quot;,
+	sw=&quot;ne&quot;,
+	w=&quot;e&quot;,
+	nw=&quot;se&quot;,
+	u=&quot;d&quot;,
+	d=&quot;u&quot;,
+	out=&quot;in&quot;,
+	[&quot;in&quot;]=&quot;out&quot;,
+}
+function mmp.astroboots(bool)
```
Comment (keneanung) [inline mudlet-mapper.xml]: Maybe rename this variable to `create`?

File: mudlet-mapper.xml [hunk 3]
```diff
@@ -8155,6 +8342,183 @@ end
...
+		end
+	end
+	removeSpecialExit(gmcp.Room.Info.num,&quot;pathfind&quot;)
+	if mmp.autowalking then
+		mmp.autowalking = false
+		mmp.gotoRoom(string.split(next(mmp.showpathcache()),&quot;_&quot;)[2])
+	end
+end
+
+function mmp.lockpaths(bool)
```
Comment (keneanung) [inline mudlet-mapper.xml]: You may want to rename this variable to `lock`.

**Comment by KaliLusternia** [comment]
I'd be happy to change some of the names up, but  I've done some other stuff on my branch and I'm not sure how to do a pull request for only the fixed up version of this stuff. :frowning_face:

**Comment by keneanung** [comment]
The following should work so we get clean PRs:
```bash
$ git stash     # if the working directory is not clean, this will save the changes
$ git checkout d0728e7dc80d652effd12c0ef1989ec295c8dc6e     # check out the commit that did not have any of these changes
$ git checkout -b separate-game-configs     # create a new branch to save things separately
$ git cherry-pick 7a52eba35a8675a5a8b3cd2e4f8bea2570983e1a    # copy the changes over into the new branch
$ git push -u origin separate-game-configs      # push the new branch to github. This would allow creating a new PR already
$ git checkout master     # go back to the branch with these changes
$ git reset --hard dd9b5e9     # remove the accidentally added commit
$ git push --force     # use the force, Anakin
$ git stash pop      # If a stash was created earlier, get it back
# change according to feedback, do commits
$ git push     # push the changes to the server
```

Afterwards you can apply any feedback to your `master` branch (make sure you are on the right branch by changing to it with `git checkout master`). The config separation changes are save in the `separate-game-configs` branch which you can activate with `git checkout separate-game-configs`.

**Pull request merged**